core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message_ru.properties

```diff
@@ -1,2 +1,11 @@
+Recommended_Java_Version_Heading=Конец поддержки Java {0} в Jenkins
```

Comment:
This always implies Java 8, doesn't it? 🤔

Reply:
I wonder if someone could run it on 9 for example... or it could be parametrized for ease of reuse of the message if one day there's another Java version migration?..